static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -1474,7 +1474,9 @@ export const findIncompatibleRules = (
       if (firstSeenError || regressionReappearedError) {
         const indices = [firstSeen, regression, reappeared, eventFrequency, userFrequency]
           .filter(idx => idx !== -1)
-          .sort();
+          .sort(function (a, b) {
+            return a - b;
+          });
```

Comment:
total and complete nit
```suggestion
          .sort((a, b) => a - b);
```

I think we can all read that OK